bytecomp/matching.ml

```diff
@@ -1540,13 +1542,24 @@ let inline_lazy_force_switch arg loc =
                sw_failaction = Some varg }, loc ))))
 
 let inline_lazy_force arg loc =
-  if !Clflags.native_code then
-    (* Lswitch generates compact and efficient native code *)
-    inline_lazy_force_switch arg loc
+  if !Clflags.afl_instrument then
+    (* Disable inlining optimisation if AFL instrumentation active,
+       so that the GC forwarding optimisation is not visible in the
+       instrumentation output. (PR#???) *)
```

Comment:
I think that https://github.com/stedolan/crowbar/issues/14 would be a good pointer to give here, in addition to GPR#1754 itself.

Reply:
Thanks! I was looking for that issue, but I thought it was on Mantis.